docs/UserGuide.md

```diff
@@ -332,17 +254,23 @@ Format: `add t/TITLE from/FROM to/TO [d/DESCRIPTION] [c/CONTACTS] [l/LOCATION]`
 * `TITLE` is the main heading of the meeting.
 * `FROM` is the start date time of the meeting in DD-MM-YYYY HH:mm format.
 * `TO` is the end date time of the meeting in DD-MM-YYYY HH:mm format.
+* `FROM` and `TO` can be in the past but must not be earlier than the year 2019.
 * `DESCRIPTION` contains additional details about the meeting, e.g. agenda.
 * `CONTACTS` represents the contacts involved in the meeting. 
 * `CONTACTS` is a comma-separated string of names.
 * `LOCATION` is the location of the meeting.
 
-<div markdown="span" class="alert alert-info">:information_source:**Note:**<br>
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes:**<br>
```

Comment:
if you check out https://ay2021s1-cs2103t-f11-2.github.io/tp/UserGuide.html#adding-a-meeting-add

this part isn't bullet point properly, mb